ArmaForces.Boderator/ArmaForces.Boderator.BotService/DataClasses/SingleUser.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+
+namespace ArmaForces.Boderator.BotService.DataClasses
+{
+    public class SingleUser
+    {
+        public ulong UserID;
+        public List<DLC> DLCList = new List<DLC>();
```

Comment:
This should be a property too. Don't use fields for publicly available stuff. And make both the property and the list readonly, e.g. `IReadOnlyList`, `IReadOnlyCollection` or maybe even `IReadOnlySet` as it should not have duplicates.

Reply:
What about adding new DLC when property is readonly? 

Or you meant something like that?
```csharp
private List<DLC> _dlcList;
public ReadOnlyCollection<DLC> DLCList {get { return _dlcList.AsReadOnly; }}
```

Reply:
Yes, almost, something like this:
```cs
private List<DLC> _dlcList;
public IReadOnlyCollection<DLC> DLCList => _dlcList.AsReadOnly();
```
We don't want to allow external modifications to any User or his data. This should be only done in `IUserRepository` or something like this, right where it's stored. So we need to always return immutable User object. Then to change anything, you must go through `IUserRepository` which will create new User instance with updated data.